src/Controllers/WishlistController.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Rapidez\MultipleWishlist\Controllers;
+
+use Illuminate\Foundation\Auth\Access\AuthorizesRequests;
+use Illuminate\Foundation\Bus\DispatchesJobs;
+use Illuminate\Foundation\Validation\ValidatesRequests;
+use Illuminate\Support\Collection;
+use Illuminate\Http\Request;
+use Illuminate\Routing\Controller;
+use Illuminate\Support\Facades\DB;
+use Rapidez\Core\Models\Product;
+use Rapidez\MultipleWishlist\Models\MagentoWishlist;
+use Rapidez\MultipleWishlist\Models\Wishlist;
+
+class WishlistController extends Controller
+{
+    use AuthorizesRequests;
+    use DispatchesJobs;
+    use ValidatesRequests;
+
+    public function index(Request $request): Collection
+    {
+        // There has to be a way to do this part more succinctly
```

Comment:
Jup: https://laravel.com/docs/master/eloquent-relationships#relationship-counting-and-custom-select-statements